grails-app/services/org/pih/warehouse/inventory/ImportPickCommand.groovy

```diff
@@ -0,0 +1,30 @@
+package org.pih.warehouse.inventory
```

Comment:
Also I just noticed that this was put in the grails-app/services directory. I don't mind it being in inventory since other stock movement classes are in there? But could this go into the grails-app/controllers directory instead?